tests-integration/tests/baseTest.ts

```diff
@@ -68,15 +80,30 @@ export const test = base.extend<TestFixtures>({
       await fs.promises.cp(logPath, logOutputPath, { recursive: true });
     }
   },
-  createProject: async ({ createTempDir }, use) => {
+  getWebview: async ({ workbox }, use) => {
```

Comment:
I'm not reviewing this as a part of this change, could you please extract it and explain what and why is happening here?

Reply:
I'll remove the integration test commit from this PR, and I'll include it after these merges are done.